examples/resources/alkira_connector_gcp_interconnect/resource.tf

```diff
@@ -11,14 +11,13 @@ resource "alkira_connector_gcp_interconnect" "example_gcp_interconnect" {
     edge_availability_domain = "AVAILABILITY_DOMAIN_1"
     customer_asn             = 56009
     bgp_auth_key             = "key"
-  }
-  segment_options {
-    segment_id               = alkira_segment.segment1.id
-    instance_name            = "instance1"
-    advertise_on_prem_routes = true
-    disable_internet_exit    = false
-    customer_gateways {
-      tunnel_count = 2
+    segment_options {
+      segment_id               = alkira_segment.segment1.id
+      advertise_on_prem_routes = true
+      advertise_default_routes = false
+      customer_gateways {
+        tunnel_count = 2
```

Comment:
This is a bit weird, it says customer_gateways, but the required field is only `tunnel_count`? How many customer_gateways we could actually associated?

Reply:
If it's only 1 block, we could consider to remove this block and change it to:

* customer_gateway_tunnel_count
* customer_gateway_loopback_ip

This way, it makes more sense than a block...

Reply:
We can have multiple `customer_gateways` .